scripts/support_bundle/README.md

```diff
@@ -0,0 +1,39 @@
+# Support Bundle Generation Script
+
+This script is to be used to collect information on your Cloud Connector Scale Set Deployment Zscaler Customer Support. The script will collect the health and scale metrics published by the Cloud Connectors, the configuration of the Scale Set deployment, and the logs from the Function App. This bundle will be used for debugging purposes to help support customer.
```

Comment:
are there any special GET privileges needed in order to connect and procure all these logs that are worth mentioning? Or should it just be "run it with the same account/Service principal you used to deploy the env and you should be good"?

Reply:
I have an action item to follow up on permissions needed for this, I will provide that update in a different PR. Thanks for calling this out.